### ISSUE TITLE
Add "point" method to Python realization of SmartProjectionPose3Factor

### ISSUE THREAD

**Issue by fatrybl** [borglab/gtsam#1784]
## Feature

Hello, there is a C++ example https://github.com/borglab/gtsam/blob/develop/examples/SFMExample_SmartFactor.cpp#L51 where the SmartProjectionPoseFactor is used. There is a SmartProjectionPoseFactor->point method which returns landmark 3D position. However, this method does not exist in Python realization of SmartProjectionPose3Factor. Could you please add it ?

## Motivation

This would make life much easier if one could get 3D position of a landmark to visualize without extra code.

**Comment by ProfFan** [comment]
You can easily add it in the `.i` files and submit a PR :)

**Comment by varunagrawal** [comment]
The `SmartProjectionPose3Factor.point` method has been wrapped and available since October 2023. @fatrybl are you using the latest version of GTSAM?

Simple example which works for me:
```python
import gtsam
measurementNoise = gtsam.noiseModel.Isotropic.Sigma(2, 1.0)
K = gtsam.Cal3_S2(50.0, 50.0, 0.0, 50.0, 50.0)
f = gtsam.SmartProjectionPose3Factor(measurementNoise, K)
f.point()
```

**Comment by fatrybl** [comment]
@varunagrawal 
Hello, i`m using gtsam 4.2. Both Importlib and poetry show it.
The provided example does not work for me.
![image](https://github.com/user-attachments/assets/fdf2f33b-8903-4a2c-a258-fdc8e1d7bc78)

**Comment by fatrybl** [comment]
The issue is no longer relevant if building the  GTSAM  with python-wrapper from source (latest current develop). Hope the fix will be added to the upcoming gtsam 4.3 release (with pip wheel)
